08-transport.Rmd

```diff
@@ -12,7 +12,9 @@ library(tidyverse)
 
 ## Introduction
 
+<!-- I would tone down, why should geographic space be more tangible in transport than in remote sensing, location analysis, ecological modeling, etc.? --> 
```

Comment:
I would tone down, why should geographic space be more tangible in transport than in remote sensing, location analysis, ecological modeling, etc.

Reply:
Fair point - as a geographer whose main area of application is transport you can understand my exuberance!